tensorflow/python/kernel_tests/math_ops/cwise_ops_unary_test.py

```diff
@@ -92,6 +92,10 @@ def _compareCpu(self, x, np_func, tf_func, grad_rtol=None, grad_atol=None):
       if x.dtype in (np.complex64, np.complex128) and tf_func == math_ops.sign:
         return  # Return early
 
+      if x.dtype in (np.float16, dtypes_lib.bfloat16.as_numpy_dtype) \
```

Comment:
Please avoid `\` for extending long lines: https://google.github.io/styleguide/pyguide.html#32-line-length

Reply:
Thanks, I removed the `\`.
I couldn't find an existing float32 test for the round op, so I added a condition check for bfloat16 in order to limit the scope of the modifications. Please let me know if any changes are needed.

Reply:
I don't understand... what does the existence of a float32 round op test have to do with skipping the gradient test for float16/bfloat16?  Does it fail if you remove this restriction?

Reply:
@cantonios Skipping the gradient test will be needed for a float32 round op test as well. It's just that there isn't one currently for float32 (https://github.com/tensorflow/tensorflow/blob/77301eaf9e579cd621f511cd8c96a85533a76628/tensorflow/python/kernel_tests/math_ops/cwise_ops_unary_test.py#L189) and I added checks for dtype to limit the scope of my changes.

I can remove the check for the datatype and just check `if tf_func == math_ops.round` if you think that's better. 
Not skipping the gradient test for round op results in errors due to reasons I mentioned in the description above.

Reply:
I see, then yes, skip for round always.

Reply:
Thanks, @cantonios. I made changes accordingly. Please re-review.